build/GenerateThemesWikiMarkdown.ps1

```diff
@@ -70,83 +75,96 @@ Function Select-ControlNameFromFile {
 }
 
 Function Read-XamlStyles {
-    Param ($xamlString, $file)
+    Param ($xamlString, $file, $xamlPath)
     [xml]$xaml = $xamlString
-    $xaml.ResourceDictionary.Style |
-    Foreach-Object { 
+    $xamlLines = Get-Content $xamlPath
+    $lineNum = 1
+    $xaml.ResourceDictionary.Style | 
+    Foreach-Object {
         Write-Output $_
+        # Get line number by Key or TargetType
+        $styleLineNumber = $null
+        $searchKey = if ($_.Key) { $_.Key } else { $_.TargetType }
+        
+        for ($i = 0; $i -lt $xamlLines.Length; $i++) {
+            if ($xamlLines[$i] -match [regex]::Escape($searchKey)) {
```

Comment:
I think `$xamlPath` is not needed as I believe you already have the content in `$xamlString` though this variable could probably be renamed to be `$xamlLines`.

Reply:
Alright, I've made the changes.